scripts/README.md

```diff
@@ -1,72 +1,118 @@
+**Step 1:** Export your sudo password as an environment variable, `SUDOPASS`. Example command:
 ```
-export SUDOPASS='xxxx'
+export SUDOPASS='your_sudo_password'
 ```
 
-2. Install git, clone this repository, fetch the `ckan_2.9_upgrade` branch, change to the scripts directory, and run the script:
-```
+**Step 2:** Install Git, clone the repository, fetch the `ckan_2.9_upgrade` branch, switch to the scripts directory, and execute the script. Follow the commands below:
+
+```bash
+# Update packages
 echo $SUDOPASS | sudo -S -k apt-get update
+
+# Install Git
 { echo $SUDOPASS; echo 'Y'; } | sudo -S -k apt-get install git
+
+# Clone the repository
 git clone https://github.com/ongov/ckanext-ontario_theme.git
 cd ckanext-ontario_theme
 
+# Fetch and checkout the branch
 git fetch origin ckan_2.9_upgrade:ckan_2.9_upgrade
 git checkout ckan_2.9_upgrade
 
+# Navigate to the scripts directory and execute the setup script
 cd scripts
 ./setup_solr.sh
 ```
 
-Note: After the tar file is extracted, there will be a message:
-`[sudo] password for ckan29: id: ‘solr’: no such user`. This is normal and can be ignored.
+Note: You may see a message stating `[sudo] password for ckan29: id: ‘solr’: no such user` after the tar file extraction. This is expected and you can ignore it.
 
-3. Unset the SUDOPASS environment variable:
+**Step 3:** Unset the `SUDOPASS` environment variable with the following command:
 ```
 unset SUDOPASS
 ```
-### Check status  
-Check the success of the installation on the command line and/or the browser.
 
-**Command line:**  
-- check that response status is 200:
+### Checking Solr Status
+You can verify the installation success via command line or a browser.
+
+- **Command Line Checks:**  
+   - Confirm response status is 200:  
+     ```
+     curl -s -o /dev/null -I -w '%{http_code}' http://{ip_address}:8983/solr/admin/cores?action=STATUS
+     ```
+   - Verify Solr node is found:  
+     ```
+     sudo /etc/init.d/solr status
+     ```
+   - Check Solr service is running:  
+     ```
+     sudo service solr status
+     ```
+
+- **Browser Checks:**  
+   - Visit `http://127.0.0.1:8983/solr/` to see if Solr is running.
+   - Check the Core Admin menu for the CKAN core with `instanceDir /var/solr/data/ckan`.
+   - Verify that the schema properties reflect the Ontario theme (Unique Key Field should be `index_id`).
+
+## Organization Data Script
+
+The `load_organization_data.py` script enables you to load organization data into your application via the CKAN API. Please fulfill the necessary prerequisites and follow the given instructions before running the script.
+
+### Prerequisites
+- Python 3.x
+- Requests library (install via `pip install requests`)
+
+### Script Setup
+1. Clone the repository containing the script.
+2. Navigate to the repository directory and install the required dependencies by running:
 ```
-$ curl -s -o /dev/null -I -w '%{http_code}' http://{ip_address}:8983/solr/admin/cores?action=STATUS
+pip
```

Comment:
typo in this script command
![image](https://github.com/ongov/ckanext-ontario_theme/assets/1310832/c5bb6394-35fd-4d33-8397-b0f38bfc54ba)

Reply:
my bad